src/MarginTrading.SettingsService.Services/MaintenanceModeService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Lykke.SlackNotifications;
+using MarginTrading.SettingsService.Core.Domain;
+using MarginTrading.SettingsService.Core.Services;
+
+namespace MarginTrading.SettingsService.Services
+{
+    public class MaintenanceModeService : IMaintenanceModeService
+    {
+        private readonly ISlackNotificationsSender _slackNotificationsSender;
```

Comment:
ISlackNotificationsSender - let's make more general sender service for this purpose, because Slack is not used in SNOW project

Reply:
Will be removed